model/controller/dual_mode_controller.cc

```diff
@@ -3471,8 +3487,8 @@ const std::unordered_map<OpCode, OpCodeIndex> DualModeController::hci_command_op
          OpCodeIndex::WRITE_SECURE_CONNECTIONS_HOST_SUPPORT},
         {OpCode::READ_AUTHENTICATED_PAYLOAD_TIMEOUT,
          OpCodeIndex::READ_AUTHENTICATED_PAYLOAD_TIMEOUT},
-        {OpCode::WRITE_AUTHENTICATED_PAYLOAD_TIMEOUT,
-         OpCodeIndex::WRITE_AUTHENTICATED_PAYLOAD_TIMEOUT},
+        // {OpCode::WRITE_AUTHENTICATED_PAYLOAD_TIMEOUT,
```

Comment:
This is not the right place to disable the command; there are two options (aside from fixing the implementation...): 
- in the list of hci_command_handlers_ below (but rootcanal will assert if the command is invoked)
- in the controller properties `model/controller/controller_properties.h`, in the supported command mask

Reply:
I've got no clear understanding of this command from reading Zephyr and other open source code. I'm not a SIG member (you probably are via Google), so I only use implementations as reference.

Disabled it the good way now